Library/Homebrew/formula_installer.rb

```diff
@@ -212,7 +212,7 @@ def install
       opoo "#{formula.full_name}: #{old_flag} was deprecated; using #{new_flag} instead!"
     end
 
-    oh1 "Installing #{Tty.green}#{formula.full_name}#{Tty.reset}" if show_header?
+    oh1 "Installing #{Formatter.success(formula.full_name)}" if show_header?
```

Comment:
Using `Formatter.success` in this and several other contexts below where it encloses a formula/dependency name feels somewhat strange to me. Should there be a dedicated formatter for these cases?